pkg/messaging/direct_messaging.go

```diff
@@ -25,6 +26,7 @@ import (
 
 const (
 	invokeRemoteRetryCount = 3
+	backoffInterval        = time.Second
```

Comment:
BTW, 1 seconds seems too long...

Reply:
For issues like the other sidecar needing to renew its certificate this isn't, which is what the unauthenticated status code check is looking for.

If it was just transient network errors I would agree this is too long.